server/middleware/office.validation.js

```diff
@@ -0,0 +1,33 @@
+const validateOffice = (req, res, next) => {
+  let verified = true;
+  const error = [];
+
+  const { type, name } = req.body;
```

Comment:
Cover this edge case --> when a user specifies an integer for type and name when trying to create an office